src/schemas/sdo/technique.schema.ts

```diff
@@ -452,10 +452,10 @@ export const techniqueSchema = attackBaseObjectSchema
       });
     }
 
-    if (x_mitre_data_sources && inMobileDomain) {
+    if (x_mitre_log_sources && inMobileDomain) {
       ctx.addIssue({
         code: z.ZodIssueCode.custom,
-        message: "x_mitre_data_sources is not supported in the 'mobile-attack' domain.",
+        message: "x_mitre_log_sources is not supported in the 'mobile-attack' domain.",
       });
```

Comment:
I'm not sure if it makes sense to rename this field to log sources. Maybe we should consider removing it altogether? The `x_mitre_data_sources` field hasn't been supported/updated for a while and was effectively replaced by "detects" relationships with Data Components. Any techniques still using this field likely contain outdated information that hasn't kept up with the current Data Sources, and this would probably become even more apparent as we transition toward Log Sources.